juice-examples/mackey-glass-rnn-regression/Cargo.toml

```diff
@@ -14,6 +14,7 @@ coaster-nn = { path = "../../coaster-nn", default-features = false , version = "
 
 csv = "1.1"
 serde = { version = "1", features = ["derive"] }
+serde_json = "1.0"
```

Comment:
nit: I had this sitting on my disk for a while https://github.com/spearow/juice/pull/190/files#diff-a8fc1a6d0bcd9cf40bbdae68476601b92bb0ba578054f13386fd31c6e0fcd201

Reply:
Can you clarify your comment? Do you mean that I shouldn't be using Serde? 

I kinda like Serde better because it can be just derived for structs, no need to write serialize/deserialize implementations. (To be fair, when you _do_ need to write them they end up pretty verbose.) And I think it's the mainstream serialization paradigm for Rust.

Capnp's main selling point is lack of serialization/deserialization step which is important for real-time apps where every bit of latency matters, which doesn't seem too important for Juice. And all those `reborrow()`s are odd :/

Reply:
They're an old relict. I don't find them to worrying though. I don't have a strong opinion.